akvo/rsr/spa/app/modules/editor/section4/section4.jsx

```diff
@@ -94,7 +94,7 @@ const Desc = ({ fields, descriptionsOrder, projectId }) => {
                         </div>
                       )}
                     >
-                      <RTE {...input} index={index} validateStatus={validateStatus} />
+                      <RTE {...input} index={index} />
```

Comment:
Why remove the `validateStatus` ?

Reply:
because it's an unused property and I can't find validationStatus being used when i was digging into it by looking for ```this.props```

https://github.com/akvo/akvo-rsr/blob/feature/4969-hierarchies-fe-remove-dependency-on-related_projects/akvo/rsr/spa/app/utils/rte.jsx#L57